frontend/src/scenes/pipeline/pipelineBatchExportConfigurationLogic.tsx

```diff
@@ -45,6 +45,9 @@ export const pipelineBatchExportConfigurationLogic = kea<pipelineBatchExportConf
         return `NEW:${service}`
     }),
     path((id) => ['scenes', 'pipeline', 'pipelineBatchExportConfigurationLogic', id]),
+    connect(() => ({
+        values: [pipelineDestinationsLogic, ['canEnableNewDestinations']],
```

Comment:
Doesn't have to be solved in this PR but connecting this logic means that it will run all the load functions in it's afterMount function which is a little unecessary  perhaps.

Reply:
yup, agreed it's unnecessary - I'll fix this in a follow-up PR